install_config/web_console_customization.adoc

```diff
@@ -335,6 +335,101 @@ endif::[]
 
 endif::[]
 
+[[configuring-catalog-categories]]
+=== Configuring Catalog Categories
+
+Catalog categories organize the display of builder images and templates on
+the Add to Project page.  A builder image or template is grouped in a category
+if it includes a tag with the same name of the category or category alias.
+Categories only display if one or more builder images or templates with matching
+tags are present in the catalog.
+
+[NOTE]
+====
+Significant customizations to the catalog categories may affect the user
+experience and should be done with careful consideration. You may need to update
+this customization in future upgrades if you modify existing category items.
+====
+
+. Create the configuration scripts within a file (for example,
+*_catalog-categories.js_*):
++
+====
+----
+// Add Go to the Languages category
+var category = window.OPENSHIFT_CONSTANTS.CATALOG_CATEGORIES.find(function(item) {
+  return item.id === "languages";
+});
+category.items.splice(2,0,{ // Insert at the third spot
+  id: "go",
```

Comment:
We should mention somewhere that `id` must be unique within the catalog. I'd probably add a comment for each property and say `id` and `label` are required. It might be easier to read if the comment was above each property.